matlab/schemas/+anatomy/AreaMask.m

```diff
@@ -0,0 +1,63 @@
+%{
+# Area mask for each scan
+-> experiment.Scan
+-> anatomy.Area
+field                    : tinyint               # slice id
+---
+mask                     : mediumblob            # mask of area
+%}
+
+classdef AreaMask < dj.Imported
+    methods(Access=protected)
+        function makeTuples(obj,key) %create clips
+            insert( obj, key );
```

Comment:
`field` is not entered (`slice_id`)   and the comment seems to not match.  This `makeTuples` seems wrong.  If it's not implemented, I would just put `error 'not implemented' rather than suggest that it does something.

Reply:
It is fine.